cucumber-junit-platform-engine/README.md

```diff
@@ -345,6 +375,13 @@ cucumber.execution.parallel.config.dynamic.factor=            # positive double.
 cucumber.execution.parallel.config.custom.class=              # class name.
                                                               # example: com.example.MyCustomParallelStrategy
 
+cucumber.execution.execution-mode.scenario=                   # same_thread or concurrent
```

Comment:
I suggested a `cucumber.execution.execution-mode.feature` because all scenarios in a feature will be executed in the same thread. Is there a reason you've changed this to scenario?

Reply:
I was thinking that naming the constant to `cucumber.execution.execution-mode.feature` will create a confusion when setting `cucumber.execution.execution-mode.feature=same_thread`, it may lead to users misinterpret that all features will run in the same thread. but below sentence makes sense

> Note that strictly speaking scenarios don't' exist. There are only pickles (both scenarios and scenario outlines are turned into pickles for individual execution).

I will rename the constant to `cucumber.execution.execution-mode.feature`. Thanks for your suggesstion!